### DISCUSSION TITLE
Add consoles' logos

### DISCUSSION THREAD

**Discussion by luchaos** [RetroAchievements/RAWeb#719] [Ideas]
When viewing a console's games it'd be neat to have the logo in the sidebar or on top.

**Comment by CoolCanuck** [comment]
Suggestion: use SVG format for the graphics :)

**Comment by oddgoo** [comment]
We could start with (and fork/credit) an open source icon repo like this one:
https://github.com/Tatohead/Console-Iconset

**Reply by televandalist** [reply]
What's the plan for the consoles that don't have icons there?

**Reply by StingX2** [reply]
Have admins commission the art team?

**Reply by luchaos** [reply]
> What's the plan for the consoles that don't have icons there?

I would assume a default icon. I used a circled play button in the v2 prototype for that.

**Reply by oddgoo** [reply]
I'm taking the responsibility to draw or find them myself, covering all current systems. I've decided to go the pixel art route since there are many more packs that are consistent in that style and can fill the gaps of the one linked above. 

I'd be happy to let the creators know and we can credit them here on the readme. I spent time finding vector ones but none were nearly as complete, consistent or good looking. 

But of course, once the base ones are in place, they can always be re-commissioned, changed, etc.

**Comment by oddgoo** [comment]
Mockup:

<img width="837" alt="image" src="https://user-images.githubusercontent.com/4585118/224517176-526ee17d-5d84-4b3b-9003-b793570a9ff6.png">

These Icons are neat enough that I could see them in the dropdown menu too:

<img width="350" alt="image" src="https://user-images.githubusercontent.com/4585118/224517209-2003cde1-02b2-466e-8ff0-7c2096820c58.png">

Makes it busier, but having the richness and personality of each console far makes up for it.

I'd be happy to tackle this

**Reply by luchaos** [reply]
I would've assumed that we'd use the actual brand logos - guess those icons work, too, though :)

**Reply by oddgoo** [reply]
> I would've assumed that we'd use the actual brand logos - guess those icons work, too, though :)

I think it's neat that all of these fit on a nice square, specially for the dropdown menu.

**Comment by oddgoo** [comment]
Pull Request: https://github.com/RetroAchievements/RAWeb/pull/1432
